sdk/tests/js/tests/tex-image-and-sub-image-2d-with-video.js

```diff
@@ -102,6 +82,10 @@ function generateTest(internalFormat, pixelFormat, pixelType, prologue, resource
                        gl.TEXTURE_CUBE_MAP_POSITIVE_Z,
                        gl.TEXTURE_CUBE_MAP_NEGATIVE_Z];
         }
+        // Handle target color space.
+        if (unpackColorSpace) {
+          gl.unpackColorSpace = unpackColorSpace;
```

Comment:
where can I find out more about this? Not much comes up when I search

https://www.google.com/search?q=%22unpackColorSpace%22

In any case, is the path of not specifying a color space still tested if color spaces are supported?

Reply:
This is to test the PR in https://github.com/KhronosGroup/WebGL/pull/3292

The unpackColorSpace is an attribute that should have a default value of 'srgb' (another test should verify that indeed this is the default value). If unpackColorSpace is not present, then the feature isn't implemented (and everything should be converted to sRGB, which is the same as setting unpackColorSpace to 'srgb').

Reply:
Maybe I missing it. To me there are 3 cases

1. set unpackColorSpace to 'srgb'
2. set unpackColorSpace to 'display-p3'
3. don't set unpackColorSpace (supposed to be the same as 1)

If 3 is not tested, how do we actually know it's the same as 1?

Reply:
re-reading your comment, I agree with you. If unpackColorSpace exists then checking it defaults to srgb should be enought

Reply:
Let's call this resolved then.